docs/docs/faq.md

```diff
@@ -165,3 +165,4 @@ If you have potential solutions or can help us move forward with updates to the
 
 [dask-tutorial]: tutorials/using_dask.md
 [selecting a profile]: https://www.nebari.dev/docs/how-tos/login-keycloak#3-selecting-a-profile
+[pytorch best practices]: https://www.nebari.dev/docs/how-tos/pytorch-best-practices
```

Comment:
```suggestion
[selecting a profile]: how-tos/login-keycloak#3-selecting-a-profile
[pytorch best practices]: how-tos/pytorch-best-practices
```

A nice-to-have, to clearly denote internal links. These will be easier to find & update if we do any restructuring in the future. :)

Reply:
Agreed.  I was following the lead of the keycloak link, but this is better.